ramalama.py

```diff
@@ -51,10 +52,17 @@ def main(args):
                            f"-v{home}:{home}",
                            "-v/tmp:/tmp",
                            f"-v{__file__}:/usr/bin/ramalama:ro",
-                           f"-v{wd}:{syspath}:ro",
-                           "-e", "RAMALAMA_HOST",
-                           "-p", f"{host}:{port}",
-                           "quay.io/ramalama/ramalama:latest", __file__] + args
+                           f"-v{wd}:{syspath}:ro"]
+            if os.path.exists("/dev/dri"):
+                conman_args += ["--device", "/dev/dri"]
+
+            if os.path.exists("/dev/kfd"):
+                conman_args += ["--device", "/dev/kfd"]
+
+            conman_args += ["-e", "RAMALAMA_HOST",
```

Comment:
This would be cleaner if  you moved these options to original conman_args

Reply:
Ok... I do need to inject the --device kargs in the middle somewhere though, I can do it just before quay.io/ramalama/ramalama:latest maybe